src/bootstrap/install.rs

```diff
@@ -210,10 +210,13 @@ install!((self, builder, _config),
         }
     };
     LlvmTools, alias = "llvm-tools", Self::should_build(_config), only_hosts: true, {
-        let tarball = builder
-            .ensure(dist::LlvmTools { target: self.target })
-            .expect("missing llvm-tools");
-        install_sh(builder, "llvm-tools", self.compiler.stage, Some(self.target), &tarball);
+        if let Some(tarball) = builder.ensure(dist::LlvmTools { target: self.target }) {
```

Comment:
I don't think this approach was right.

See https://github.com/rust-lang/rust/pull/109196#discussion_r1138336815

Reply:
@ozkanonur Thank you for fixing.